### ISSUE TITLE
`create-yoshi-app` .vscode `launch.json` is broken

### ISSUE THREAD

**Issue by hepiyellow** [wix/yoshi#575]
In all templates, the `.vscode.json` looks something like this:
```
{
  "version": "0.2.0",
  "configurations": [{
    "name": "Launch Mocha",
    "type": "node",
    "request": "launch",
    "program": "${workspaceRoot}/node_modules/yoshi/debug/mocha",
    "cwd": "${workspaceRoot}"
  }]
}
```

While `node_modules/yoshi/debug/mocha` does not exist, and the launch config should be somerthing like this (e.g. for protractor):
```
"configurations": [
    {
      "name": "Unit Test Debug",
      "type": "node",
      "request": "launch",
      "args": [
        "test",
        "--jest",
        "--debug-brk"
      ],
      "port": 9229,
      "program": "${workspaceFolder}/node_modules/.bin/yoshi"
    },
    {
      "name": "e2e Test Debug",
      "type": "node",
      "request": "launch",
      "args" : ["test", "--protractor", "--debug-brk"],
      "port": 9229,
      "program": "${workspaceFolder}/node_modules/.bin/yoshi"
    }
  ]
```

**Comment by ranyitz** [comment]
Thanks @hepiyellow, will you have the time to send a PR that fixes it?

**Comment by hepiyellow** [comment]
Sure.
Could you just tell me what is the cli command option for running test with puppeteer?
I don't see it in the [docs](https://github.com/wix/yoshi/blob/master/docs/faq/YOSHI-CLI.md)

**Comment by ranyitz** [comment]
1. `puppeteer` is not a test runner, which means you can run tests in `jasmine`/`mocha`/`jest` and `require('puppeteer')` from them. If you're using [`jest-yoshi-preset`](https://github.com/wix/yoshi/tree/master/packages/jest-yoshi-preset) you would be able to run only the [puppeteer environment](https://github.com/wix/yoshi/tree/master/packages/jest-yoshi-preset#puppeteer-environment) using `MATCH_ENV=e2e` ([learn more](https://github.com/wix/yoshi/tree/master/packages/jest-yoshi-preset#usage))

2. You can choose it yourself, but if you're using [yoshi's protractor configuration](https://github.com/wix/yoshi/blob/36d4156774eaa414dda69e0dc8f852c09955f71a/packages/yoshi/config/protractor.conf.js#L27) it's Jasmine.

**Comment by hepiyellow** [comment]
no time, sorry.

**Comment by netanelgilad** [comment]
The generated `launch.json`s were fixed by #633. They are now generated for jest testing. Since we want to move all generated projected to work with `jest` and `puppeteer` we are not generating a launch config for protractor. @ranyitz correct me if I'm wrong.

I think we can close this issue. @hepiyellow objections/thoughts?

**Comment by hepiyellow** [comment]
Sure, we can close this. Thanks !

BTW: about #633,
just a little "optimization",
I think the Launch config names should be shorter, because , in VS code, the launch config dropdown is not so wide usually.
And if the names are short, you can see the current config without clicking it.

So I'd :
- [ ] remove the work "Test" from all names, since "Jest" or "E2E" implies already that it is a test.
- [ ] I would have the name start with either "Jest" "E2E".
